addon/adapters/rest.js

```diff
@@ -1228,7 +1228,8 @@ function ajaxError(adapter, payload, requestData, responseData) {
   } else if (responseData.textStatus === 'timeout') {
     error = new TimeoutError();
   } else if (responseData.textStatus === 'abort' || responseData.status === 0) {
-    error = new AbortError();
+    // TODO REVIEW should we use an adapter method for this, e.g. like handleResponse?
```

Comment:
Any ideas on this topic, should the ability to handle the abort error live on the adapter?

```js
try {
  error = adapter.handleAbort(
    responseData.status,
    responseData.headers,
    payload || responseData.errorThrown,
    requestData
  );
} catch (e) {
  error = e;
}
```